### PR TITLE
Use C# 7.

### PR DESCRIPTION
**Pull request by ejball**

### CODE REVIEW

**Comment by ejball** [comment]
@ddunkin, you added a "unix" build but I don't have an easy way to test it. Do you have a `.travis.yml` file that would work? Either way, any objection to this pull request?

**Comment by ddunkin** [comment]
There's a `.travis.yml` on master now.

**Comment by ejball** [comment]
Thanks!!

File: src/Facility.Definition/Fsd/FsdParser.cs [hunk 1]
```diff
@@ -18,9 +18,7 @@ public sealed class FsdParser
 		/// <exception cref="ServiceDefinitionException">Thrown if the parse fails.</exception>
 		public ServiceInfo ParseDefinition(NamedText source)
 		{
-			ServiceInfo service;
-			IReadOnlyList<ServiceDefinitionError> errors;
-			if (TryParseDefinition(source, out service, out errors))
+			if (TryParseDefinition(source, out var service, out var errors))
```
Comment (ddunkin) [inline src/Facility.Definition/Fsd/FsdParser.cs:21]: This is the one new C# 7 feature I don't like. It makes it hard to see where variables are declared, and the scoping is not what I expect. I do like being able to use `var` for out parameters, though. I'm sure I'll get used to it eventually.

**Comment by ddunkin** [comment]
👍

File: src/Facility.Definition/Fsd/FsdParser.cs (hunk 1 above)
Comment (ejball) [reply src/Facility.Definition/Fsd/FsdParser.cs:21]: Yeah, I mostly just do it because ReSharper tells me to. Probably not the best reason. :-)

**Pull request merged**